pkg/apis/serving/v1beta1/inference_service_validation.go

```diff
@@ -141,6 +146,40 @@ func validateInferenceService(isvc *InferenceService) (admission.Warnings, error
 	return allWarnings, nil
 }
 
+// Validate multi-node variables(tensor-parallel-size, pipeline-parallel-size, WorkerSpec.Size)
```

Comment:
```suggestion
// validateMultiNodeVariables can be one of (tensor-parallel-size, pipeline-parallel-size, WorkerSpec.Size)
```

Reply:
I don't quite understand the suggestion, so I'll rephrase it myself to make it clearer

Reply:
it is to keep the Golang comments pattern, `// method/field name description`

Reply:
I'll update it to match the style of the other comments in the file.